### PR TITLE
Add touch examples

### PR DESCRIPTION
**Pull request by ionciubotaru**
Adding example folder will help other users to implement new specific ESP32 functions

### CODE REVIEW

**Comment by me-no-dev** [comment]
ok, this is totally wrong in Arduino perspective.
Examples for ESP32 need to go into ```libraries/ESP32/examples/``` and folder name MUST match the .ino name. ```libraries/ESP32/``` need to have the proper files in order to be discoverable in Arduino IDE

**Comment by ionciubotaru** [comment]
fixed

**Comment by me-no-dev** [comment]
see [here](https://github.com/esp8266/Arduino/tree/master/libraries/esp8266) how the folder should look like ;) needs to have the src folder, library.properties and so on

**Comment by me-no-dev** [comment]
also please capitalize the name of the sketch (all sketches start with capital letter)

File: libraries/ESP32/examples/Touch/touchIntrerrupt/touchIntrrerupt.ino [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+void gotTouch1(){
+ touch2detected = true;
+}
+
+void setup() {
+  Serial.begin(115200);
+  delay(1000); // give me time to bring up serial monitor
+  printf("\n ESP32 Touch Interrupt Test\n");
```
Comment (me-no-dev) [inline libraries/ESP32/examples/Touch/touchIntrerrupt/touchIntrrerupt.ino]: this should be ```Serial.println("\nESP32 Touch Interrupt Test");```

Comment (ionciubotaru) [inline libraries/ESP32/examples/Touch/touchIntrerrupt/touchIntrrerupt.ino]: fixed

**Comment by ionciubotaru** [comment]
fixed all sugestions

**Comment by me-no-dev** [comment]
Please please please do pay attention at what you are doing! Do test the sketches that you commit! folder and sketch name are all wrong....

**Comment by me-no-dev** [comment]
```TouchInterrupt``` is how it MUST be spelled

**Comment by ionciubotaru** [comment]
Sorry for mistake, I correct it.
I test the schetch before upload, but with a different name

**Pull request merged**